components/integrations/bots/add_bot/add_bot.jsx

```diff
@@ -0,0 +1,516 @@
+                        onSubmit={this.handleSubmit}
+                    >
+                        <div className='form-group'>
+                            <label
+                                className='control-label col-sm-4'
+                                htmlFor='username'
+                            >
+                                <FormattedMessage
+                                    id='bots.add.username'
+                                    defaultMessage='Username'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8'>
+                                <input
+                                    id='username'
+                                    type='text'
+                                    maxLength='64'
+                                    className='form-control'
+                                    value={this.state.username}
+                                    onChange={this.updateUsername}
+                                />
+                                <div className='form__help'>
+                                    <FormattedMessage
+                                        id='bot.add.username.help'
+                                        defaultMessage='Usernames must be all lowercase. They cannot be longer than 21 characters and can only contain letters, numbers, periods, hyphens, and underscores.'
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div className='form-group'>
+                            <label
+                                className='control-label col-sm-4'
+                                htmlFor='boticon'
+                            >
+                                <FormattedMessage
+                                    id='bots.add.icon'
+                                    defaultMessage='Bot Icon'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8'>
+                                <div className={'bot-img-container'}>
+                                    <img
+                                        className={'bot-img'}
+                                        alt={'bot image'}
+                                        src={imageURL}
+                                        style={imageStyles}
+                                    />
+                                    {removeImageIcon}
+                                </div>
+                                <div
+                                    className='btn btn-sm btn-primary btn-file sel-btn'
+                                >
+                                    <FormattedMessage
+                                        id='setting_picture.select'
+                                        defaultMessage='Select'
```

Comment:
```suggestion
                                        defaultMessage='Upload an image'
```

Reply:
@crspeller I assume this would change the string everywhere else in the UI (since its ID is `setting_picture.select`). Is it used elsewhere other than Account Settings > General > Profile Picture?

Reply:
No not sure how the automatic i18n stuff would handle this. I will update the text and the ID.